### DISCUSSION TITLE
How is database generated?

### DISCUSSION THREAD

**Discussion by tempelmann** [kishikawakatsumi/applelocalization-web#131] [General]
I would like to understand where the database is getting the data from. I had a look at the [applelocalization-tools](https://github.com/kishikawakatsumi/applelocalization-tools) code but I don't understand what it does.

Would you please give an overview how you collect the data, and from where?

**Comment by kishikawakatsumi** [comment]
It scans the `/System/Library` and `/Developer` directories and retrieves strings from the `*.strings` or `*.loctable` files of bundles such as frameworks, apps, etc. contained there. What exactly do you not understand?

**Reply by tempelmann** [reply]
Thank you.
There are also Apple's downloadable glossaries that could be used. And some .xib files contain translations, too. There are also Apple-made apps inside /Applications that could be scanned. So, there are a lot of ways, and that's what I was interested in.
